test/tokens/PErc20.sol

```diff
@@ -2,10 +2,10 @@
 
 pragma solidity 0.8.13;
 
-import "./IPErc20.sol";
+import './IPErc20.sol';
 
 /**
- * @dev Implementation of the {IERC20} interface.
+ * @dev Implementation of the {ERC20} interface, with additions
```

Comment:
yeah, but it isn't. 

this whole chain of dependencies is a victim of O.Z. stupidity and I don't want to rewrite the whole thing as auditors like the OZ bs. This thing actually is just an expanded ERC20 - it has their specific additions to the "standard".

its named PErc20 to denote that it's the basis for the 'permissioned' ERC2612, as the next line of comments explain

it could have been OZERC  or EERC but we used PErc20 and its gunna stay that way, OZs dependency chain is super-fucked and i'm not opening that box again